app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -93,9 +99,62 @@ class TopicInfoFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicInfoFragment_loadFragment_checkDefaultTopicDescriptionLines_fiveLinesVisible() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.topic_description_text_view)).check(matches(maxLines(/* lineCount= */ 5)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_seeMoreIsVisible() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.see_more_text_view)).check(matches(isDisplayed()))
+      onView(withId(R.id.see_more_text_view)).check(matches(withText(R.string.see_more)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_clickSeeMore_textChangesToSeeLess() {
+    launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
+      onView(withId(R.id.see_more_text_view)).perform(click())
+      onView(withId(R.id.see_more_text_view)).check(matches(withText(R.string.see_less)))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_clickSeeMore_topicDescriptionLinesChangeToSix() {
```

Comment:
This testcase is failing.
<img width="1639" alt="Screenshot 2020-03-30 at 11 42 23 AM" src="https://user-images.githubusercontent.com/16301028/77881052-b0254d80-727b-11ea-890e-89cd79fcac63.png">